scripts/dataset-download.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# shellcheck disable=SC2153
```

Comment:
I'm guessing this should most likely be on the specific line, rather than full file.